test/compile/mark/bar.test.ts

```diff
@@ -42,7 +42,12 @@ describe('Mark: Bar', function() {
         "y": {"bin": true, "type": "quantitative", "field": "IMDB_Rating"},
         "x": {"scale": {"type": 'log'}, "type": "quantitative", "field": 'US_Gross', "aggregate": "mean"}
       },
-      "data": {"url": 'data/movies.json'}
+      "data": {"url": 'data/movies.json'},
```

Comment:
You're fixing default behavior in `config.ts`, but here modify the test to no longer test the default behavior.  The correct fix is to fix the asserts below.